xbout/load.py

```diff
@@ -439,21 +439,26 @@ def _read_splitting(filepath, info=True):
     ds = xr.open_dataset(str(filepath))
 
     # Account for case of no parallelisation, when nxpe etc won't be in dataset
-    def get_scalar(ds, key, default=1, info=True):
+    def get_scalar(ds, key, default=1, only_positive=False, info=True):
```

Comment:
Looks like we only ever actually call this with `only_positive=True`. Maybe it's worth just renaming the function to `get_nonnegative_scalar` and removing the argument?

Reply:
Hmm yeah, the only other place it might have been used is in`utils._separate_metadata`.